lib/python/qmk/cli/generate/version_h.py

```diff
@@ -48,6 +51,7 @@ def generate_version_h(cli):
         f"""
 #define QMK_VERSION "{git_version}"
 #define QMK_BUILDDATE "{current_time}"
+#define QMK_BCD_VERSION {git_bcd_version}
```

Comment:
Should be aligned to other defines
```suggestion
#define QMK_VERSION_BCD {git_bcd_version}
```